libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -199,9 +199,9 @@ void GCS_MAVLINK::send_battery_status(const AP_BattMonitor &battery, const uint8
                                     MAV_BATTERY_FUNCTION_UNKNOWN, // function
                                     MAV_BATTERY_TYPE_UNKNOWN, // type
                                     got_temperature ? ((int16_t) (temp * 100)) : INT16_MAX, // temperature. INT16_MAX if unknown
-                                    battery.get_cell_voltages(instance).cells, // cell voltages
-                                    battery.has_current(instance) ? battery.current_amps(instance) * 100 : -1, // current
-                                    battery.has_current(instance) ? battery.current_total_mah(instance) : -1, // total current
+                                    battery.get_cell_voltages(instance).cells, // cell voltages in mV
+                                    battery.has_current(instance) ? battery.current_amps(instance) * 100 : -1, // current in cA
```

Comment:
Stuff like this is an absolutely awful way to note units, IMO. This is what I'm referring to, it should either be written out fully (or in this case can probably be ignored since this is easy to read). In general I think adding the .'s/capitalization in a bunch of places has done nothing to improve the readability and in a lot of ways made it worse.